test/unit/gateways/stripe_test.rb

```diff
@@ -439,6 +439,17 @@ def test_client_data_submitted_with_purchase_without_email_or_order
     end.respond_with(successful_purchase_response)
   end
 
+  def test_client_data_submitted_with_metadata_in_options
+    stub_comms(@gateway, :ssl_request) do
+      updated_options = @options.merge({:metadata => {:option1 => 'optional', :option2 => 'not required'}, :order_id => "42", :email => "foo@wonderfullyfakedomain.com"})
```

Comment:
Could you name these keys something that's super obviously a test key/not expected in any way?